target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -21,6 +21,12 @@ case "$board" in
 "glinet,ar150")
 	ucidef_set_led_wlan "wlan" "WLAN" "gl-ar150:orange:wlan" "phy0tpt"
 	;;
+"tplink,tl-wdr3600"|\
+"tplink,tl-wdr4300")
+	ucidef_set_led_usbdev "usb1" "USB1" "tp-link:green:usb1" "1-1.1"
+	ucidef_set_led_usbdev "usb2" "USB2" "tp-link:green:usb2" "1-1.2"
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "tp-link:green:wlan2g" "phy0tpt"
```

Comment:
this should also be possible to encode inside OF. if not let me know and we'll add support

Reply:
With "trigger-sources" as for usb it is not possible.
For now i set the wireless leds with "$(get_dt_led wifi)" and an alias for the led.
Would be great if it will also be possible with "trigger-sources".